include/core/truthTable/pla_parser.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "core/truthTable/truth_table.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] inline syrec::TruthTable::Cube::Value transform_pla_to_constants(const char& c) {
+        syrec::TruthTable::Cube::Value constant;
+        syrec::TruthTable::Cube::Value one  = true;
+        syrec::TruthTable::Cube::Value zero = false;
+        switch (c) {
+            case '-':
+            case '~':
+                return constant;
+
+            case '0':
+                return zero;
+            case '1':
+                return one;
+
+            default:
+                assert(false);
+                return constant;
+        }
+    }
```

Comment:
Remove the `[[maybe unused]]`.

This function can be simplified. See the below suggestion.
```suggestion
    inline TruthTable::Cube::Value transform_pla_to_constants(const char& c) {
        switch (c) {
            case '-':
            case '~':
                return {};
            case '0':
                return false;
            case '1':
                return true;
            default:
                assert(false);
                return {};
        }
    }
```

Note that the `assert(false)` only triggers in debug mode and not in release configurations. If you want to indicate an error, then throw an exception.